packages/synapse-react-client/public/mockServiceWorker.js

```diff
@@ -2,7 +2,7 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (1.2.2).
+ * Mock Service Worker (1.3.2).
  * @see https://github.com/mswjs/msw
  * - Please do NOT modify this file.
```

Comment:
Was this bumped by the msw upgrade? Just curious given the file's request to not modify the file directly!